region/crateria/west.json

```diff
@@ -231,59 +249,128 @@
           "to": [
             {
               "id": 8,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             }
           ]
         },
         {
           "from": 2,
           "to": [
             {
-              "id": 7,
-              "requires": [
-                {"or" : [
-                  {"enemyDamage": {
-                    "enemy": "Green Space Pirate (standing)",
-                    "type": "contact",
-                    "hits": 3
-                  }},
-                  {"enemyKill":{
-                    "enemies": [
-                      [
-                        "Green Space Pirate (standing)",
-                        "Green Space Pirate (standing)",
-                        "Green Space Pirate (standing)"
-                      ],
-                      [
-                        "Green Space Pirate (standing)",
-                        "Green Space Pirate (standing)"
+              "id": 4,
+              "strats": [
+                {
+                  "name": "Tank the Damage",
+                  "notable": false,
+                  "requires": [
+                    {"enemyDamage": {
+                      "enemy": "Green Space Pirate (standing)",
+                      "type": "contact",
+                      "hits": 3
+                    }}
+                  ]
+                },
+                {
+                  "name": "Kill the Pirates",
+                  "notable": false,
+                  "requires": [
+                    {"enemyKill":{
+                      "enemies": [
+                        [
+                          "Green Space Pirate (standing)",
+                          "Green Space Pirate (standing)",
+                          "Green Space Pirate (standing)"
+                        ],
+                        [
+                          "Green Space Pirate (standing)",
+                          "Green Space Pirate (standing)"
```

Comment:
what's with the grouping? ammo drops?

Reply:
See documentation for enemyKill: https://github.com/miketrethewey/sm-json-data/blob/master/logicalRequirements.md

Essentially, the groups are an estimation of what can be caught together by a PB blast.